yajsapi/utils/index.ts

```diff
@@ -6,13 +6,13 @@ import Callable from "./callable";
 import CancellationToken from "./cancellationToken";
 import eventLoop from "./eventLoop";
 import getAllProperties from "./getAllProperties";
-import logger, * as logUtils from "./log";
-import { changeLogLevel } from "./log";
```

Comment:
Seems like `utils.changeLogLevel("debug");` is not avaliable anymore - I think it is good idea to do not change interface, or correct examples

Reply:
Yes, you're right. I forgot about examples. Now logger is a param of executor constructor and has method setLevel to change log level. For the default logger (winston for nodejs), when the param is null, we may add logLevel parameter to the executor constructor. I'll fix it.

Reply:
@Braunmann please review again